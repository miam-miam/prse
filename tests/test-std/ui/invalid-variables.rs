use prse::parse;

fn main() {
    let l = "test: 5";

    parse!(l, "test: {foo()}");
    parse!(l, "test: {-8}");
    parse!(l, "test: {256}")
}
