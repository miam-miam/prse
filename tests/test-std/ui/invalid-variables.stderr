error: expected identifier
 --> ui/invalid-variables.rs:6:5
  |
6 |     parse!(l, "test: {foo()}");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `parse` (in Nightly builds, run with -Z macro-backtrace for more info)

error: unexpected end of input, position must be between 0 and 255.
 --> ui/invalid-variables.rs:7:5
  |
7 |     parse!(l, "test: {-8}");
  |     ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `parse` (in Nightly builds, run with -Z macro-backtrace for more info)

error: unexpected end of input, position must be between 0 and 255.
 --> ui/invalid-variables.rs:8:5
  |
8 |     parse!(l, "test: {256}")
  |     ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `parse` (in Nightly builds, run with -Z macro-backtrace for more info)
